### PR TITLE
Ported PicklesHelper to FAKE 5

### PR DESCRIPTION
**Pull request by magicmonty**
Ported PicklesHelper to Fake.Tools.Pickles (see http://www.picklesdoc.com for description of functionality).

### CODE REVIEW

File: src/app/Fake.Tools.Pickles/Pickles.fs [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+    
+ type DocumentationFormat =
+    | DHTML
+    | HTML
+    | Word
+    | JSON
+    | Excel
+
+/// The Pickles parameter type
+[<CLIMutable>]
```
Comment (matthid) [inline src/app/Fake.Tools.Pickles/Pickles.fs]: please remove climutable

File: src/app/Fake.Tools.Pickles/Pickles.fs [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+      SystemUnderTestVersion: string option
+      /// Maximum time to allow xUnit to run before being killed.
+      TimeOut : TimeSpan
+      /// Option which allows to specify if failure of pickles should break the build.
+      ErrorLevel : ErrorLevel
+      /// Option which allows to enable some experimental features
+      IncludeExperimentalFeatures : bool option
+    }
+
+let currentDirectory = Directory.GetCurrentDirectory()
```
Comment (matthid) [inline src/app/Fake.Tools.Pickles/Pickles.fs]: we should at least make that internal or private

File: src/app/Fake.Tools.Pickles/Pickles.fs [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+/// - `OutputDirectory` - `currentDirectory @@ "Documentation"`
+/// - `OutputFileFormat` - `DHTML`
+/// - `TestResultsFormat` - `Nunit`
+/// - `LinkedTestResultFiles` - []
+/// - `SystemUnderTestName` - `None`
+/// - `SystemUnderTestVersion` - `None`
+/// - `TimeOut` - 5 minutes
+/// - `ErrorLevel` - `Error`
+/// - `IncludeExperimentalFeatures` - `None` 
+let PicklesDefaults =
```
Comment (matthid) [inline src/app/Fake.Tools.Pickles/Pickles.fs]: probably best to change to private or internal as well?

**Comment by magicmonty** [comment]
Thanks for the review. I overlooked these

**Comment by matthid** [comment]
Thanks!

**Pull request merged**